sycl/source/detail/graph_impl.hpp

```diff
@@ -350,6 +350,17 @@ class graph_impl {
     if (PropList.has_property<property::graph::no_cycle_check>()) {
       MSkipCycleChecks = true;
     }
+    if (SyclDevice.get_info<
+            ext::oneapi::experimental::info::device::graph_support>() ==
+        info::graph_support_level::unsupported) {
+      std::stringstream ss;
+      ss << SyclDevice.get_backend();
+      std::string backendString = ss.str();
+      throw sycl::exception(
+          sycl::make_error_code(errc::invalid),
+          backendString +
+              " backend is not yet supported by SYCL Graph extension.");
```

Comment:
```suggestion
      std::stringstream Stream;
      Stream << SyclDevice.get_backend();
      std::string BackendString = Stream.str();
      throw sycl::exception(
          sycl::make_error_code(errc::invalid),
          BackendString +
              " backend is not yet supported by SYCL Graph extension.");
```

Local variable names should follow clang formatting guidelines as well.